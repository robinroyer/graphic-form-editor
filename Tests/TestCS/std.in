S s00 0 0 5 5
R r01 10 10 5 5
C c01 10 10 5
L l01 2 2 3 3
PL pl01 1 1 2 2 3 3 4 4
S s01 0 0 20 20
EXIT