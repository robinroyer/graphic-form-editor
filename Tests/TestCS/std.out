OK
#New object: s00
OK
#New object: r01
OK
#New object: c01
OK
#New object: l01
OK
#New object: pl01
OK
#New object: s01
